packages/inter-protocol/src/proposals/add-auction.js

```diff
@@ -179,6 +190,7 @@ export const addAuction = async ({
   );
 
   auctionUpgradeNewInstance.resolve(governedInstance);
+  newContractGovBundleId.resolve(contractGovernorBundle.bundleID);
```

Comment:
When we make the governor for the auctioneer that we're adding, we don't seem to use an installation based on this ID.

Is there a way to test that we got that part right?

Reply:
> When we make the governor for the auctioneer that we're adding, we don't seem to use an installation based on this ID.

`startInstance()` wants an `Installation`, while `upgradeContract()` wants a `BundleId`. The cleanest way of tying those together seems to be in `getManifestForAddAuction()`.

That takes the bundleRef (`contractGovernorRef`) from the proposalBuilder script and passes both it and the installation produced from it (`installations: { contractGovernor: restoreRef(contractGovernorRef) }`) to the auction proposal. The addAuction proposal uses the installation to start the new auction, and passes the bundleId through promise space to the `upgrade-vaults` proposal.

I could verify that both governors have the same bundle, but I don't see how the Installation and bundleId could become dissociated.